### PR TITLE
Added yum --exclude option as parameter.

### PR DESCRIPTION
**Pull request by tweisshaar**
I need to exclude certain packages while updating the servers. Here is a small patch that works for me.

### CODE REVIEW

File: packaging/os/yum.py [hunk 1]
```diff
@@ -805,6 +817,7 @@ def main():
             disable_gpg_check=dict(required=False, default="no", type='bool'),
             # this should not be needed, but exists as a failsafe
             install_repoquery=dict(required=False, default="yes", type='bool'),
+            exclude=dict(),
```
Comment (resmo) [inline packaging/os/yum.py]: IMHO this should be `exclude=dict(type='list', default=[])` and then use `yum_exclude_cmd = "--exclude=%s" % ','.join(exclude)` for passing it as cli argument.

Comment (tweisshaar) [inline packaging/os/yum.py]: Thank you for your review! Works perfectly.

**Comment by resmo** [comment]
Okay nice! Please commit and push the changes, so the pull request gets updated.

**Comment by mpdehaan** [comment]
Can you please add  a 'version_added: 1.9' in the DOCUMENTATION block?  Thanks!

**Comment by mwhahaha** [comment]
@mpdehaan - Any progress on getting this merged? I also need this.

**Comment by gregdek** [comment]
Hi @tweisshaar. Thanks for submitting this pull request. Please accept our apologies for taking so long to review it.

Unfortunately, due to this delay, it appears that the underlying code has changed, and this PR needs to be rebased. If you believe that this PR still applies, please rebase and we will then put it into our (new, faster) community review process as soon as possible.  Again, apologies for the delay and the extra work.

**Comment by retr0h** [comment]
@tweisshaar any chance of a rebase?

**Comment by gregdek** [comment]
@tweisshaar Note that this functionality has been included through another patch in 2.0 -- it was a patch that many submitted. :) Thanks for your PR.

**Pull request closed without merging**